synthetic_signwriting/generator.py

```diff
@@ -241,6 +241,9 @@ def render_fsw(self):
             generated_left_hand = str(hex(keyframe.left_hand_signwriting))[2:]
             generated_face = '2ff00'     # TODO adding synthetic face option
 
+            # adding 8 offset to the left hand
+            generated_left_hand = str(hex(int(generated_left_hand, 16) + 0x8))[2:]
```

Comment:
this does not belong here.
it belongs at lines 86~ 
if *not* right hand, then mirror the symbol (add 0x8)

Reply:
you right, I updated it in the if-else condition